frontend/app/lib/sprites/tyton.js

```diff
@@ -77,6 +86,30 @@ class Tyton extends Phaser.Sprite {
     this.start_speaking(strings);
   }
 
+  succeed() {
+    this.health += 10;
+    this.power += 10;
+  }
+
+  fail() {
+    let damage = 10;
+
+    if (this.health > damage) {
+      this.health -= damage;
+    }
+    else {
```

Comment:
this should be in the `update` function and should be conditioned on `this.health <= 0`